test/fixtures/multipleErrors.json

```diff
@@ -0,0 +1,33 @@
+[
+	{"type": "google.rpc.BadRequest", "value":{
```

Comment:
nit: can you please format this JSON file?  There are several ways to do it, the easiest one is `cat multipleErrors.json | jq . > tmp && mv tmp multipleErrors.json`, alternatively, in Node.js, do `JSON.stringify(content, null, '  ')`, or just format the file in VS Code (Ctrl+Shift+I).